profiles-sdk-examples/cdata-connection/README.md

```diff
@@ -0,0 +1,97 @@
+# CData Connection
+
+This example is a CLI application for reading data from a CData Cortex Connection and writing said data to a separate
+Connection. This builds off the [Local Clients](../local-clients/README.md) example for its initial setup, but uses a
+separate set of connections defined in [cdata-connections.yml](../main-app/src/main/resources/spec/cdata-connections.yml).
+
+Two source Connections are defined in the [Local Catalog](../local-clients/README.md#catalog):
+- `cdata-csv` - Connection to a local `members_100_v14.csv` file. This will be used for the following examples.
+- `cdata-bigquery` - CData Connection to Google BigQuery. To use this you will need to update the `url` parameter to
+  your BigQuery data and update the `query` parameter accordingly. See [Notes on CData BigQuery Connection](#notes-on-cdata-bigquery-connection)
```

Comment:
Yeah i put "accordingly" because a user needs to update the query in correspondence/accordance with the url. (A query is made on the data at the URL, and the result of that query is the Connection data. The query needs to be based on the data at the URL and might have column names and details specific to the data). 

I think I can just say "... update the `url` and `query` parameters to match your BigQuery data"